src/components/ui/Image/Image.tsx

```diff
@@ -1,23 +1,59 @@
 import { useGetThumborImageData } from '@vtex/gatsby-plugin-thumbor'
 import { GatsbyImage } from 'gatsby-plugin-image'
 import React, { useMemo } from 'react'
-import imagesConf from 'src/images/config'
+import type { ThumborImageOptions } from '@vtex/gatsby-plugin-thumbor'
 import type { GatsbyImageProps } from 'gatsby-plugin-image'
 
 interface Props extends Omit<GatsbyImageProps, 'image'> {
-  variant: string
-  src: string
+  image: ThumborImageOptions
 }
 
-function Image({ variant, src, ...imgProps }: Props) {
```

Comment:
Does Thumbor offer a playground so people can preview and learn what each of these options does? If it does, I think it's worth referencing it here.

Reply:
These are not thumbor props, but gatsby image ones. The thumbor one are those passed in options, like `fit-in` properties